src/lib.rs

```diff
@@ -94,6 +94,7 @@ pub mod properties;
 mod slice_transform;
 mod snapshot;
 mod sst_file_writer;
+pub mod statistics;
```

Comment:
Should this module be public?

Reply:
Yes. This is how API users can access `Ticker` enum.